README.md

```diff
@@ -18,3 +18,90 @@ yarn run build
+    
+* [AdGuard Dutch filter](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/DutchFilter/sections/adservers.txt)
+      
+* [AdGuard Dutch filter first-party servers](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/DutchFilter/sections/adservers_firstparty.txt)
+
+* [EasyList](https://raw.githubusercontent.com/easylist/easylist/master/easylist/easylist_adservers.txt)
+    
+* [EasyList third-party servers](https://raw.githubusercontent.com/easylist/easylist/master/easylist/easylist_thirdparty.txt)
+    
+* [EasyList Germany](https://raw.githubusercontent.com/easylist/easylistgermany/master/easylistgermany/easylistgermany_adservers.txt)
+    
+* [EasyList Italy](https://raw.githubusercontent.com/easylist/easylistitaly/master/easylistitaly/easylistitaly_adservers.txt)
+
+* [EasyList China](https://raw.githubusercontent.com/easylist/easylistchina/master/easylistchina.txt)
+    
+* [ABPindo](https://raw.githubusercontent.com/ABPindo/indonesianadblockrules/master/src/advert/adservers.txt)
+    
+* [ABPindo third-party servers](https://raw.githubusercontent.com/ABPindo/indonesianadblockrules/master/src/advert/thirdparty.txt)
+    
+* [AdblockID](https://raw.githubusercontent.com/realodix/AdBlockID/master/src/adservers.adfl)
+    
+* [hostsVN](https://raw.githubusercontent.com/bigdargon/hostsVN/master/filters/adservers.txt)
+###  Tracking servers
+#### These filter lists block domains that are used for tracking and analytics
+    
+* [AdGuard Tracking Protection filter third-party trackers](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/SpywareFilter/sections/tracking_servers.txt)
+    
+* [AdGuard Tracking Protection filter first-party trackers](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/SpywareFilter/sections/tracking_servers_firstparty.txt)
+     
+* [AdGuard Tracking Protection filter mobile trackers](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/SpywareFilter/sections/mobile.txt)
+    
+* [EasyPrivacy](https://raw.githubusercontent.com/easylist/easylist/master/easyprivacy/easyprivacy_trackingservers.txt)
+    
+* [EasyPrivacy third-party trackers](https://raw.githubusercontent.com/easylist/easylist/master/easyprivacy/easyprivacy_thirdparty.txt)
+    
+* [EasyPrivacy international trackers](https://raw.githubusercontent.com/easylist/easylist/master/easyprivacy/easyprivacy_trackingservers_international.txt)
+     
+* [EasyPrivacy third-party international](https://raw.githubusercontent.com/easylist/easylist/master/easyprivacy/easyprivacy_thirdparty_international.txt)
+###  Cryptominers
+#### These filter lists are used against cryptominers
+
+* [AdGuard Base filter cryptominers](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/BaseFilter/sections/cryptominers.txt)
+###  Exception
+#### This list is used to exclude domains that cause serious site breakages
```

Comment:
These lists are used to ...

Also, it's not clear to me what's the difference between them. Should we explain?

Reply:
They provide the same function, I don't think there is any need to explain separately 
@zloyden @Alex-302 What do you think?

Reply:
@vbagirov they do the same goal, but approach to the goal is different.